tests/test_benchmark.py

```diff
@@ -220,22 +200,19 @@ def benchmark(
     v0: NDArray,
     box: NDArray,
     bound_potentials: List[BoundPotential],
-    hmr: bool = True,
+    dt: float = 1.5e-3,
```

Comment:
Realized I change the output of `python tests/test_benchmark.py` so a lot of things now use 1.5fs rather than 2.5. Changing back so benchmarking stays consistent.